07-performance.Rmd

```diff
@@ -49,17 +49,20 @@ highlight standard tricks or alternative methods that may improve performance.
 
 ### The `if` vs `ifelse` functions {-}
 
-The `ifelse` function 
+`ifelse()` is a vectorised version of the standard control-flow function `if(test) if_yes... else if_no...` that works as follows:
+
 ```{r eval=FALSE}
-ifelse(test, yes, no)
+ifelse(test, if_yes, if_no)
 ```
-is a vectorised version of the standard control-flow `if` function. The return value
-of `ifelse` is filled with elements from the `yes` and `no` arguments that are
+
+In the above imaginary example, the return value
+of  is filled with elements from the `yes` and `no` arguments that are
```

Comment:
This  should be `is_yes` and `is_no`. Also, would `if_true` and `if_false` be better?